customer-commons/src/test/resources/features/update_customer.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Update an existing Customer
+
```

Comment:
Again, background would make this less DRY-incompatible :)

Reply:
How much would you push into Background? I am now using Administrator and User to show that user has valid credentials or not.

Reply:
Anything that is repeated in multiple features and not directly related to the state of the particular scenario can be put in the Background.